components/nexus-mime/src/main/resources/builtin-mimetypes.properties

```diff
@@ -30,3 +30,4 @@ tgz = application/x-tgz,application/x-gzip,application/gzip
 bz2 = application/x-bzip2
 tbz = application/x-bzip2,application/x-bzip
 nupkg = application/zip
+kar = application/zip,application/x-zip-compressed
```

Comment:
btw, any particular reason why this entry also has `x-zip-compressed` but nupkg doesn't?